api/FileReaderSync.json

```diff
@@ -325,7 +325,7 @@
               "version_added": "8"
             },
             "ie": {
-              "version_added": true
+              "version_added": false
```

Comment:
@vinyldarkscratch did you mean to leave this change? (This is the `worker_support` entry.)

Reply:
I'm wondering why my git client's "revert" isn't working fully, hmm...